docs/src/token.md

```diff
@@ -2,9 +2,76 @@
 title: Token Program
 ---
 
-An ERC20-like Token program on the Solana blockchain.
+A Fungible Token program on the Solana blockchain.
 
-The project comprises:
+This program provides an interface and implementation that third parties can use to create and trade their tokens.
```

Comment:
```suggestion
This program provides an interface and implementation that third parties can use to create and use their tokens.
```

Reply:
Maybe make the first `use` => `utilize`?